style.css

```diff
@@ -13,56 +13,135 @@
 body {
   background: #e5e5e5;
   font-family: "Poppins", sans-serif;
-  min-width: 768px;
 }
 
-.body-white {
-  width: 60%;
-  background: #f7f7f9;
+header {
+  padding: 1rem;
+  justify-content: space-between;
+  background-color: white;
 }
 
-.header {
+.header-1 {
   display: flex;
-  width: 27.4375rem;
-  background: white;
+  flex-direction: column;
+  align-content: flex-start;
+  padding: 3rem 1rem 3rem 1rem;
+  margin: 8rem 1.5rem 2rem 1.5rem;
+}
+
+.header {
+  background-color: white;
   background-image: url(images/bg-image.png);
   background-repeat: no-repeat;
-  background-position: center;
-  padding: 29% 25px;
+  background-size: 100%;
   border-bottom-left-radius: 70px;
-  align-items: center;
+  padding: 5rem 1rem 2rem 1rem;
+
+  /* border: solid green 2px; */
 }
 
-.logo {
+.navbar {
   display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 130%;
 }
 
-.logo a {
+.logo {
   text-decoration: none;
   font-weight: 700;
   font-size: 18px;
   line-height: 16px;
   color: #6070ff;
-  width: 5rem;
+  margin-top: 8px;
 }
 
-.navbar {
-  position: fixed;
+.btn a {
   display: flex;
-  justify-content: space-between;
-  height: 50px;
-  background-color: white;
+  text-decoration: none;
+  color: var(--primary-b-500, #396df2);
+  text-align: center;
+  font-size: 1.3rem;
+  font-weight: 500;
+}
+
+.nav-links li {
+  list-style: none;
 }
 
 .nav-links {
-  display: none;
+  position: fixed;
+  left: -100%;
+  top: 0;
+  flex-direction: column;
+  background-color: #7f8cff;
+  width: 100%;
+  text-align: center;
+  height: 100vh;
+  opacity: 0.93;
+}
```

Comment:
- [ ] Kindly make sure your implementation of the design matches the one shown in the Figma design as I noticed some discrepancies in the design of the mobile menu. As you can see in the Figma design below the menu links are aligned to the left and there's a blur effect on the background the background color also has a darker tone to it so I'd suggest you update your design to match these.
- Figma design:
  ![image](https://github.com/jerymey/Portfolio-setup/assets/104968236/a96e8d8a-0550-4f3e-8bec-05ef0df56b5d)
- Your design:
  ![image](https://github.com/jerymey/Portfolio-setup/assets/104968236/b0089229-a092-4535-ac24-4937ba0ebc39)